sale_resource_booking/wizards/resource_booking_sale.py

```diff
@@ -9,7 +9,7 @@ class ResourceBookingSale(models.TransientModel):
     _name = "resource.booking.sale"
     _description = "Sale order generator for resource booking types"
 
-    type_id = fields.Many2one(
+    rb_type_id = fields.Many2one(
```

Comment:
suggestion: a more expressive name:
```suggestion
    booking_type_id = fields.Many2one(
```

Reply:
Anyway, this can be considered more a bug in `sale_order_type`, not cleaning the context (I assume with `default_type_id`). Changing one field name means that a migration script should be provided, so we should avoid it, or provide the script.

Reply:
Well, this is a transient model, so maybe is not such a big deal

Reply:
OK, being a transient, no migration script indeed, but anyway, the problem will arise with any transient using `type_id`, so I insist that should be fixed in `sale_order_type`.

Reply:
I find it totally necessary to change the name of the field of the sale_order_type module and the purchase_order_type. They are "common" names that can be stepped on as has happened with different modules.
But I would make the change for the v.17 migration. How do you see it?

Reply:
In any case, I also think it is necessary to rename the field and avoid using common names. So I have changed the name as suggested by @chienandalu. 
What do you think?